MediaBrowser.Api/UserService.cs

```diff
@@ -395,10 +394,11 @@ public object Post(AuthenticateUser request)
                 throw new MethodNotAllowedException("Hashed-only passwords are not valid for this API.");
             }
 
+            // password should always be null
             return Post(new AuthenticateUserByName
             {
                 Username = user.Name,
-                Password = null, // This should always be null
```

Comment:
Why move this comment?

Reply:
Inline comments have a habit of running past the maximum length, so I prefer to leave them on their own line.

Reply:
Ok, I just find them clearer in some situations